lua/packer/load.lua

```diff
@@ -103,6 +103,7 @@ packer_load = function(names, cause, plugins)
   elseif cause.event then
     cmd(fmt('doautocmd <nomodeline> %s', cause.event))
   elseif cause.ft then
+    cmd(fmt('doautocmd <nomodeline> FileType %s', cause.ft))
     cmd(fmt('doautocmd <nomodeline> %s FileType %s', 'filetypeplugin', cause.ft))
```

Comment:
As @Iron-E noted, if we're doing a group-less `FileType`  the other two `FileType` triggers become redundant. Would you mind removing them?

Reply:
Done!